src/configs/status.rs

```diff
@@ -7,13 +7,17 @@ use starship_module_config_derive::ModuleConfig;
 pub struct StatusConfig<'a> {
     pub format: &'a str,
     pub symbol: &'a str,
+    pub sucess_symbol: &'a str,
```

Comment:
Looks like we've got a typo here. 😄 
```suggestion
    pub success_symbol: &'a str,
```

Reply:
Should be fixed :)